### PR TITLE
fix: edit device and pagination in address book report

### PR DESCRIPTION
**Pull request by vivekp8000**
## Change Description
  fix: edit device and csv issue fixed

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/reports/address-book/address-book/address-book.component.ts [hunk 1]
```diff
@@ -26,33 +26,27 @@ export class AddressBookComponent implements OnInit {
   this.fetchAddressBook()
   }
   
-  async fetchAddressBook(refresh?:boolean) {
-    if (refresh === true) {
```
Comment (kunalfleethawks) [inline src/app/pages/reports/address-book/address-book/address-book.component.ts:30]: Why this code was removed?

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please explain the changes

File: src/app/pages/reports/address-book/address-book/address-book.component.ts (hunk 1 above)
Comment (vivekp8000) [reply src/app/pages/reports/address-book/address-book/address-book.component.ts:30]: i just checked that. so pagination is working fine on ip, cloud and devapp.

**Comment by vivekp8000** [comment]
> Please explain the changes
edit device title appears when we update a device and pagination is working fine in address book.

**Pull request merged**